inst/tests/tests.Rraw

```diff
@@ -3569,7 +3570,31 @@ DT[,`:=`(last.x=tail(x,1L),last.x1=tail(x1,1L)),by=y]
 test(1086, class(DT$last.x), c("POSIXct", "POSIXt"))
 test(1087, class(DT$last.x1), "ITime")
 
-# Tests 1088-1093 were non-ASCII. Now in DtNonAsciiTests
+# chmatch on 'unknown' encoding (e.g. as.character(as.symbol("\u00E4")) )falling back to match, #2538 and #4818
+x1 = c("al\u00E4", "ala", "\u00E4allc", "coep")
+x2 = c("ala", "al\u00E4")
+tstc = function(y) unlist(lapply(y, function(x) as.character(as.name(x))), use.names=FALSE)
+test(1088.1, requires_utf8="\u00E4", chmatch(x1, x2), match(x1, x2)) # should not fallback to "match"
```

Comment:
maybe easier to understand/write tests as `requires_utf8=c(x1, x2)`?

also, maybe better as a `local()` check here too, to reduce the visual noise of all the identical `requires_utf8=` inputs?